frontend/src/container/CreateAlertV2/EvaluationSettings/TimeInput/TimeInput.tsx

```diff
@@ -0,0 +1,155 @@
+import './TimeInput.scss';
+
+import { Input } from 'antd';
+import React, { useEffect, useState } from 'react';
+
+export interface TimeInputProps {
+	value?: string; // Format: "HH:MM:SS"
+	onChange?: (value: string) => void;
+	disabled?: boolean;
+	className?: string;
+}
+
+function TimeInput({
```

Comment:
- Could this be part of component library?
- Is there no component available in component library for this?

Reply:
Don't think there is another exact component like this.
There is the TimePicker from antd that we can use. 
https://ant.design/components/time-picker
Do you think this is a better alternative?